src/components/inline-link/types.ts

```diff
@@ -3,10 +3,12 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+import type { ComponentProps } from 'react'
+import NextLink from 'next/link'
 import { LinkProps } from 'components/link'
 import { TextProps } from 'components/text'
 
-export interface InlineLinkProps {
+export interface InlineLinkProps extends ComponentProps<typeof NextLink> {
```

Comment:
We should extend `LinkProps` here since that's the component being interacted with. It already handles extending `ComponentProps<typeof NextLink>` for us! 😊

```suggestion
export interface InlineLinkProps extends LinkProps {
```

Also curious what you think about leveraging the `opensInNewTab` prop instead of using `target="_blank"`?

Reply:
Cool, I'll update this to extend `LinkProps`. Using the `target` attribute feels more natural to me but do you have thoughts on the `opensInNewTab` props?

Reply:
I do! 😊 Have my thoughts in a reply here in the `Link` component's RFC: https://docs.google.com/document/d/14embplHRyd2zO1A9a7Zws75xZeNZ6yG6WaXr57DnSuQ/edit?disco=AAAAijiamkE

Reply:
Thanks for linking to that doc, I'm going to retain the use of `target` here for now.